charts/trino/Chart.yaml

```diff
@@ -15,9 +15,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.2.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 351
+appVersion: "355"
+
+icon: https://trino.io/assets/images/trino-logo/trino-ko_tiny-alt.svg
+
+maintainers:
```

Comment:
Let's leave this out, since this chart is maintained by the Trino community. We avoid having specific authors listed in various places (in contrast to, for example, projects that list authors in each file) as that has social implications and can make people hesitant to make changes.

Instead, we can do
```yaml
home: https://trino.io/
sources:
  - https://github.com/trinodb/charts
  - https://github.com/trinodb/trino/tree/master/core/docker

Reply:
I added it because of default helm lint require it to be there, fortunately there is a flag `--validate-maintainers=false`

Reply:
We coud make this
```yaml
maintainers:
  - name: Trino community